tests/test_auth.py

```diff
@@ -0,0 +1,39 @@
+""" Tests for auth/views.py methods """
+import pytest
+
+from flask import g
+from http import HTTPStatus
+
+from tests import factories
+
+
+def test_activate_unauthenticated_get(client):
+    """ Tests if unauthenticated GET to activate returns 405"""
+    response = client.get("/auth/activate")
+    assert response.status_code ==  HTTPStatus.METHOD_NOT_ALLOWED
+
+
+def test_activate_unauthenticated(client):
+    """ Tests if unauthenticated POST to activate returns correct screen"""
+    response = client.post("/auth/activate")
+    assert b"<h1>You are not logged in</h1>" in response.data
+    assert response.status_code ==  HTTPStatus.OK
+
+
+@pytest.mark.skip(reason="Test must set user in session")
```

Comment:
@paulodamaso 
```python
    employee = factories.EmployeeFactory(
        company=factories.CompanyFactory(),
        ....
    )

    with client.session_transaction() as session:
        session["user_id"] = employee.id
```

Reply:
@vryazanov Didn't worked, unfortunately :(. It shows the following message:
```
AttributeError: 'SecureCookieSession' object has no attribute 'commit'
```